### PR TITLE
Switch in contracts to verify in `core`

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

**Comment by mihails-strasuns-sociomantic** [comment]
Ping :)

File: src/ocean/core/Enum.d [hunk 1]
```diff
@@ -127,15 +127,8 @@
...
-
-/***************************************************************************
-
-    Imports
-
-***************************************************************************/
-
 import ocean.transition;
-
+import ocean.core.Test;
```
Comment (nemanja-boric-sociomantic) [inline src/ocean/core/Enum.d]: This _should_ be another commit, but I don't think it matters much, up to you.

File: src/ocean/core/ByteSwap.d [hunk 2]
```diff
@@ -103,7 +104,7 @@ struct ByteSwap
 
     final static void swap16 (void *dst, size_t bytes)
     {
-        assert ((bytes & 0x01) is 0);
+        verify((bytes & 0x01) is 0);
```
Comment (gavin-norman-sociomantic) [inline src/ocean/core/ByteSwap.d:107]: The commit message isn't really true. This commit alters more than just "in contracts".

File: src/ocean/core/Enum.d [hunk 3]
```diff
@@ -727,45 +720,40 @@ version ( UnitTest )
...
-    {
-        assert(names.length == values.length);
-        assert(names.length);
-    }
-    body
     {
-        // Internal name/value lists
-        assert(E.names == names);
-        assert(E.values == values);
+        test(names.length == values.length);
```
Comment (gavin-norman-sociomantic) [inline src/ocean/core/Enum.d:726]: Separate commit for this?

File: src/ocean/core/MessageFiber.d [hunk 4]
```diff
@@ -483,6 +470,11 @@ class MessageFiber
     }
     body
     {
+        verify(
```
Comment (gavin-norman-sociomantic) [inline src/ocean/core/MessageFiber.d:473]: This is a very important one! :+1:

File: src/ocean/core/SmartEnum.d [hunk 5]
```diff
@@ -111,19 +111,13 @@
 
 module ocean.core.SmartEnum;
 
-
-/*******************************************************************************
```
Comment (gavin-norman-sociomantic) [inline src/ocean/core/SmartEnum.d]: Why all the mixed in formatting changes? :(

Comment (mihails-strasuns-sociomantic) [reply src/ocean/core/SmartEnum.d]: Because I couldn't decide where to put new import without it :trollface:

**Comment by mihails-strasuns-sociomantic** [comment]
Updated with test changes split into separate commit

File: src/ocean/core/SmartEnum.d (hunk 5 above)
Comment (gavin-norman-sociomantic) [reply src/ocean/core/SmartEnum.d]: Hmm... I don't see the relation.

**Comment by mihails-strasuns-sociomantic** [comment]
Rebased

**Review by gavin-norman-sociomantic — APPROVED** [review]
